src/components/MasterDetails.js

```diff
@@ -167,6 +169,9 @@ export default class MasterDetails extends React.Component<MasterDetailsDefaultP
     const tableWidth = this.props.split;
     const detailsWidth = 12 - tableWidth;
     const halfPadding = `${this.props.padding / 2}px`;
+    const detailsRow = this.props.rows.find((row) => {
+      return row.id === this.state.detailsRow.id;
```

Comment:
`detailsRow` should be removed from the state and replaced with a string called `detailsRowId` since the Id is the only part of it that we ever use